Procfile

```diff
@@ -1 +1 @@
-web: node app.js
+web: node server.js
```

Comment:
the run command should be updated in the `package.json` file as well

Reply:
Yes, good catch, will change